api/management/api.go

```diff
@@ -343,6 +355,22 @@ type IdentityProviderRepresentation struct {
 	TrustEmail                *bool                   `json:"trustEmail,omitempty"`
 }
 
+// ThemeConfiguration struct
+type ThemeConfiguration struct {
+	Settings *configuration.ThemeConfigurationSettings `json:"settings,omitempty"`
+	Logo     *string                                   `json:"logo,omitempty"`
+	Favicon  *string                                   `json:"favicon,omitempty"`
+}
+
+// UpdatableThemeConfiguration struct
+type UpdatableThemeConfiguration struct {
+	ThemeName    *string                                   `json:"theme_name,omitempty"`
```

Comment:
We could have a unique struct ThemConfiguration with the ThemeName, Settings, Translations, Logo and FavIcon... (which are "nilable")
UpdatableXXX struct where introduced in some of our interfaces because we add to face an issue when some users of the interface where transmitting null values in the JSON to tell they wanted no change in an update statement when we were using to null value to clear a field in database. We don't have this problem here